syntax = "proto3";

package sift.common.type.v1;

enum ChannelDataType {
  CHANNEL_DATA_TYPE_UNSPECIFIED = 0 [deprecated = true];
  CHANNEL_DATA_TYPE_DOUBLE = 1;
  CHANNEL_DATA_TYPE_STRING = 2;
  CHANNEL_DATA_TYPE_ENUM = 3;
  CHANNEL_DATA_TYPE_BIT_FIELD = 4;
  CHANNEL_DATA_TYPE_BOOL = 5;
  CHANNEL_DATA_TYPE_FLOAT = 6;
  CHANNEL_DATA_TYPE_INT_32 = 7;
  CHANNEL_DATA_TYPE_UINT_32 = 8;
  CHANNEL_DATA_TYPE_INT_64 = 9;
  CHANNEL_DATA_TYPE_UINT_64 = 10;
}
